taskfile.yaml

```diff
@@ -6,6 +6,12 @@ dotenv:
 
 interval: "500ms"
 
+vars:
+  PATH_CHARM: "./charm/charm_amd64.charm"
+  IMAGE: "specs.canonical.com:latest"
+  APP_NAME: "specs"
+  ROCK: "canonical-specs-v2_2.0.0_amd64.rock"
```

Comment:
Isn't the version number `v2_2.0.0` here subject to change as this app evolves? If the version number if bumped, would you expect that this var should also be bumped, or that the task invoker must override this?

I suspect it may be better to target the `canonical-specs-v*.rock` that was just built so we don't need to maintain this version manually